src/pages/AdvancedOrders/index.tsx

```diff
@@ -27,11 +27,10 @@ export default function AdvancedOrdersPage() {
 
   return (
     <>
-      <CreatedInOrderBookOrdersUpdater />
       <FillAdvancedOrdersDerivedStateUpdater />
       <styledEl.PageWrapper isUnlocked={isUnlocked}>
         <styledEl.PrimaryWrapper>
-          <AdvancedOrdersWidget>
+          <AdvancedOrdersWidget updaters={<TwapUpdaters />}>
```

Comment:
Question: would it work if you just pass the component import rather than "instantiating" it here, like this:

```suggestion
          <AdvancedOrdersWidget updaters={TwapUpdaters}>
```

I haven't tested, just curiosity

Reply:
<img width="960" alt="image" src="https://github.com/cowprotocol/cowswap/assets/7122625/9e9a57b9-1edc-4c24-bd44-1b540bf2d80b">

Nope